woocommerce/woocommerce
# Fix issue with changelog generation selection case-sensitivity

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the changelog auto-generation to also allow for upper-cased checks (`[X]`), as they are permitted by GitHub.

Closes #46933.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Verify that the change file is properly created in this test PR https://github.com/woocommerce/woocommerce/pull/46937 based on the PR body.
2. Verify that the newly added tests in this PR cover the upper-case scenario and that tests pass.
3.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
This PR only affects tooling and therefore a changelog entry is not required.
</details>
